### ISSUE TITLE
Yo can you ban this goofy ass kid

### ISSUE THREAD

**Issue by NotNimu** [ShadowMario/FNF-PsychEngine#10927]
### What is your question?

![37D99DF9-BF35-403D-B65C-80BD2E7BF4AF](https://user-images.githubusercontent.com/115769608/197079029-6835201b-75d6-48ed-842f-3ff47023ea7f.jpeg)
Bros 9ft under 💀💀💀💀💀💀💀💀💀💀💀

**Comment by MemeHoovy** [comment]
yes, @ShadowMario please do so (I know his notifications are off, but still)

**Comment by NotNimu** [comment]
> yes, @ShadowMario please do so (I know his notifications are off, but still)

Mf your committing ban speedrun any% 4:67

**Comment by MemeHoovy** [comment]
how is mentioning him gonna get me banned? he has notifications off so mentioning him will do pretty much nothing 🤷🏻‍♂️

**Comment by NotNimu** [comment]
> how is mentioning him gonna get me banned? he has notifications off so mentioning him will do pretty much nothing 🤷🏻‍♂️

Many people have gotten banned before just by pinging him, I recommend not doing it next time.

**Comment by MemeHoovy** [comment]
again, he has **notifications off** I highly doubt he would

**Comment by NotNimu** [comment]
> again, he has **notifications off** I highly doubt he would

It’s still talking to a brick wall tho, pinging it is basically punching it.

**Comment by MemeHoovy** [comment]
fine, I won't do it again

**Comment by CaptainBaldi** [comment]
> ### What is your question?
> ![37D99DF9-BF35-403D-B65C-80BD2E7BF4AF](https://user-images.githubusercontent.com/115769608/197079029-6835201b-75d6-48ed-842f-3ff47023ea7f.jpeg) Bros 9ft under 💀💀💀💀💀💀💀💀💀💀💀

YOU GUYS ARE FUCKING BULLYING ME FUCK YOU GUYS RATS

**Comment by MemeHoovy** [comment]
> broooo, this guy who made this issue is a hypocrite himself, he broke like 6 tos rules in a span of a month lmao.

really now?

**Comment by NotNimu** [comment]
> > broooo, this guy who made this issue is a hypocrite himself, he broke like 6 tos rules in a span of a month lmao.
> 
> really?
 yea like half of the shit that happened in this repo was in my hands, thekillera1 almost all of the spam nimu incidents, all me.

**Comment by NotNimu** [comment]
> broooo, this guy who made this issue is a hypocrite himself, he broke like 6 tos rules in a span of a month lmao.

Hypocrite to hypocrite conversation

**Comment by MemeHoovy** [comment]
> > broooo, this guy who made this issue is a hypocrite himself, he broke like 6 tos rules in a span of a month lmao.
> 
> Hypocrite to hypocrite conversation

your not even worth insulting

**Comment by NotNimu** [comment]
> bro, keep making goofy ahhh shitposts, but Jimmy; You used a SLUR on me, you admitted to being an owner of this account, and you try to hide. Jimmy, fuck you, I honestly thought you were cool smh
Y’know what, maybe I should rethink my actions
![84EE258E-8C65-4C11-9DFB-70EE7A0B6412](https://user-images.githubusercontent.com/115769608/197087402-3a75331d-caa6-48f7-a0a0-59a60e295500.gif)

**Comment by TheKitBoi** [comment]
I got blocked by him

**Comment by NotNimu** [comment]
> > I got blocked by him
> 
> also, this bitch is a minor by the way, wanting nudes of an adult online video game character which isn't real.

Look into my eyes and tell me if you know what irony is

**Comment by NotNimu** [comment]
> That also harasses people on the site.

triple trouble if it was good

**Comment by NotNimu** [comment]
> That also harasses people on the site.

he said "i got blocked by him" by the guy with the ms paint dog icon, not me.

**Comment by NotNimu** [comment]
> > > > That also harasses people on the site.
> > > 
> > > 
> > > he said "i got blocked by him" by the guy with the ms paint dog icon, not me.
> > 
> > 
> > Hey Jimmy, the trolling is starting to become not revalent, nobody will barely respond to it, to only give you criticism. Nobody is starting to care, you basically lost who you were on social media here.
> 
> You deleted your account for absolutely nothing, just over funny little jokes that will die to become not popular. Never will be popular again, now you have your porn addiction. To wanting nudes of a character, for your own pleasure.
> 
> Jimmy, haven't you realized? People might had looked up to you as one point an actually funny guy. But NO.
> 
> You did this, because the "Trolling & Porn" consumed you todo this, you ditched your account for this, over a trolling and porn wanting of a virtual character.

i feel like I’m being mugged
![6750F2B3-843E-41E8-9E4E-838245386298](https://user-images.githubusercontent.com/115769608/197366345-0c8b98c0-2f27-4077-b360-fc20fa7db4f8.jpeg)

**Comment by BeastlyGhost** [comment]
yall are dense asf Shadow has github pings off lmao

**Comment by ShadowMario** [comment]
> yall are dense asf Shadow has github pings off lmao

^ THIS IS 100% TRUE

You didn't have to make an issue about it, don't do it again or you will also get banned.

**Comment by BeastlyGhost** [comment]
> > > yall are dense asf Shadow has github pings off lmao
> > 
> > 
> > ^ THIS IS 100% TRUE
> > You didn't have to make an issue about it, don't do it again or you will also get banned.
> 
> Hes a troll that will harass anyone bro.

tá mas onde isso afeta o grêmio

**Comment by Gazozoz** [comment]
isso rebaixa eles para a serie B

**Comment by MemeHoovy** [comment]
> isso rebaixa eles para a serie B


https://user-images.githubusercontent.com/97995998/197658684-4dc6a70f-9d95-4113-9829-2750bf6795dd.mp4
